spec.html

```diff
@@ -43057,12 +43058,21 @@ <h1>
           1. Let _value_ be IteratorValue(_result_).
           1. IfAbruptRejectPromise(_value_, _promiseCapability_).
           1. Let _valueWrapper_ be PromiseResolve(%Promise%, _value_).
+          1. If _valueWrapper_ is an abrupt completion, and _done_ is not *true*, then
```

Comment:
```suggestion
          1. If _valueWrapper_ is an abrupt completion and _done_ is *false*, then
```

IteratorComplete coerces its result to a boolean (if it doesn't throw), so "not true" and "false" are equivalent.